### PR TITLE
Add go.mod

### PR DESCRIPTION
**Pull request by s3rj1k**
Signed-off-by: s3rj1k <evasive.gyron@gmail.com>

### CODE REVIEW

**Comment by s3rj1k** [comment]
@ramr Would be nice to have this merged :)

**Comment by s3rj1k** [comment]
Also please tag a release after merge

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module github.com/ramr/go-reaper
+
+go 1.16
```
Comment (ramr) [inline go.mod:3]: 1.16 is wee bit too recent. Maybe set it a few years back ala 1.12?  Thx

**Review by ramr — CHANGES_REQUESTED** [review]
Happy to merge it in and create a release - maybe set version to 1.1[2-3]. Thx

File: go.mod (hunk 1 above)
Comment (s3rj1k) [reply go.mod:3]: That version actually does not do anything for older golang versions.
And I am pretty sure that code would still compile for older versions.

Comment (s3rj1k) [reply go.mod:3]: How about a bit more recent that 1.12, something like 1.14?
go1.14 (released 2020/02/25)

Comment (ramr) [reply go.mod:3]: Sure that works. Thx

Comment (ramr) [reply go.mod:3]: Actually don't bother changing it. Have a few spare cycles now (before dinner!!). 
I'll merge it and lower the version.

**Comment by ramr** [comment]
Thanks for the PR. 

Merged and tagged release - https://github.com/ramr/go-reaper/releases/tag/0.2.1

File: go.mod (hunk 1 above)
Comment (s3rj1k) [reply go.mod:3]: Thanks!

**Pull request merged**